tests/pytorch_tests/model_tests/base_pytorch_test.py

```diff
@@ -34,21 +36,34 @@ def __init__(self, unit_test, float_reconstruction_error=1e-7):
         super().__init__(unit_test)
         self.float_reconstruction_error = float_reconstruction_error
 
+    def get_fw_hw_model(self):
+        return {
+            'no_quantization': generate_fhw_model_pytorch(name="no_quant_pytorch_test",
+                                                          hardware_model=generate_test_hw_model({'weights_n_bits': 32,
+                                                                                                 'activation_n_bits': 32})),
+            'all_32bit': generate_fhw_model_pytorch(name="32_quant_pytorch_test",
+                                                    hardware_model=generate_test_hw_model({'weights_n_bits': 32,
+                                                                                           'activation_n_bits': 32})),
+            'all_4bit': generate_fhw_model_pytorch(name="4_quant_pytorch_test",
+                                                   hardware_model=generate_test_hw_model({'weights_n_bits': 4,
+                                                                                          'activation_n_bits': 4})),
+        }
+
     def get_quantization_configs(self):
         return {
             'no_quantization': mct.QuantizationConfig(mct.QuantizationErrorMethod.NOCLIPPING,
                                                       mct.QuantizationErrorMethod.NOCLIPPING,
-                                                      32, 32, False, True, True,
+                                                      False, True, True,
                                                       enable_weights_quantization=False,
                                                       enable_activation_quantization=False),
             'all_32bit': mct.QuantizationConfig(mct.QuantizationErrorMethod.NOCLIPPING,
                                                 mct.QuantizationErrorMethod.NOCLIPPING,
-                                                32, 32, False, True, True,
+                                                False, True, True,
                                                 enable_weights_quantization=True,
                                                 enable_activation_quantization=True),
             'all_4bit': mct.QuantizationConfig(mct.QuantizationErrorMethod.NOCLIPPING,
                                                mct.QuantizationErrorMethod.NOCLIPPING,
-                                               4, 4, False, False, True,
+                                               False, False, True,
```

Comment:
We can remove this method and refactor Pytorch tests to run only over different hw models (as all configs here are identical). But let's do it in a different PR. For now, it's ok. Just add a TODO please